### DISCUSSION TITLE
Will this work with K9CNF or MR7VT?

### DISCUSSION THREAD

**Discussion by kabone** [danielewood/sierra-wireless-modems#106] [Q&A] [answered]
I wanted to buy another one of these and I placed an order on ebay from china. They emailed and said they can do faster shipping if I would do K9CNF or MR7VT? Will that matter for running this script?

Thanks

**Comment by danielewood** [comment, marked as answer]
Probably wont make a difference. As far as I can tell, those two are just more Dell/Lenovo variants. Most likely they use the same USB identifiers. There may be band support differences, but that is all down to target markets.

**Reply by kabone** [reply]
Thanks for letting me know. I figured that was the case since the link you included in the readme to eBay includes those models, but figured I'd ask.
